test/suite/test_rollback_to_stable19.py

```diff
@@ -37,7 +37,7 @@ def timestamp_str(t):
     return '%x' % t
 
 # test_rollback_to_stable19.py
-# Test that rollback to stable aborts both insert and remove updates from a single prepared transaction
+# Test that rollback to stable does not open any dhandles that don't have unstable updates.
```

Comment:
The wrong test is updated, this change must be for the `20.py` file.

Reply:
Fixed